src/main/resources/org/jenkinsci/plugins/discardbuild/DiscardBuildPublisher/help.jelly

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    <p>
+        Discard Old Build plugin functionality is detailed below. The plugin allows for setting build discard
+        conditions with greater specificity than is available in the Jenkin's core function. It is recommended that
+        users carefully test plugin configurations before implementing in important projects to avoid any plugin
+        misconfigurations producing unwanted build history discards.
+    </p>
+    <dl>
+        <dt><code>Days to keep builds</code></dt>
```

Comment:
You can [create individual help files](https://wiki.jenkins.io/display/JENKINS/Basic+guide+to+Jelly+usage+in+Jenkins#BasicguidetoJellyusageinJenkins-Helpfiles) for each field. Create a file named after the field for example `src/main/resources/org/jenkinsci/plugins/discardbuild/DiscardBuildPublisher/help-daysToKeep.html` and put the help in it:

```html
<div>
    Sets the amount of days a build will be kept before being discarded by the plugin. Leaving this entry empty disables the corresponding discard action.
</div>
```

Reply:
I have moved the help documentation over to separate files for each field. I may be misunderstanding this, but checkboxes do not seem to be declared as _fields_ in the Jelly configuration. I'm unsure how to produce the ? box to the right of checkbox tables, which this new feature along with some additional existing ones (Status to discard and Keep last builds) use. Are you aware how I can do this? Using what is described here only produces an interface effect for the other features.

Reply:
I will look for similar implementations in existing plugins, but you may know how to produce this more quickly than I can research it.